.envrc.example

```diff
@@ -1 +1,7 @@
-export PORT=3000
\ No newline at end of file
+export PORT=3000
+export MAILGUN_SMTP_PORT=587
+export MAILGUN_SMTP_SERVER=smtp.mailgun.org
+export MAILGUN_SMTP_LOGIN=postmaster@sandboxd.mailgun.org
+export MAILGUN_SMTP_PASSWORD=Password
+MAILGUN_API_KEY=Key
+MAILGUN_PUBLIC_KEY=Key
```

Comment:
Let's remove these last two improperly set up MAILGUN lines. They need to be prefixed with export to work.

In fact, you can remove all these MAILGUN settings from your .envrc and the .envrc.example. They aren't needed in development; only on Heroku, which has environment variables set differently.